api-mobile/app.ts

```diff
@@ -11,7 +11,8 @@ const defaultLog = getLogger('app');
 
 import { authenticate } from './src/utils/auth-utils';
 
-const HOST = process.env.API_HOST || 'localhost:3002';
+const HOST = process.env.API_HOST || 'localhost';
+const PORT = Number(process.env.API_PORT || '3002');
```

Comment:
Why aren't we simply using 3001? We are in a different POD.

Reply:
Mainly so that if you run it locally, it doesn't clash with the other api